### PR TITLE
Get rid of some `#[allow(rustc::untranslatable_diagnostic)]`

### PR DESCRIPTION
**Pull request by GrigorenkoPV**
@rustbot label +A-translation
cc https://github.com/rust-lang/rust/issues/100717

### CODE REVIEW

**Comment by GrigorenkoPV** [comment]
Very strange, I will revert the last commit (51190638ab2ecb98aeb3976fc394ac3ff2a28551)

**Comment by GrigorenkoPV** [comment]
Oh, reverted the wrong thing. It should have been 18128d062e6d9ed03c1981f09ad1aa6557338df7

**Comment by nnethercote** [comment]
Nice to see so many `FIXME` comments removed :)

@bors r+

File: tests/ui/coroutine/gen_block.none.stderr [hunk 1]
```diff
@@ -73,7 +73,6 @@ LL |     let _ = || yield true;
    = note: see issue #43122 <https://github.com/rust-lang/rust/issues/43122> for more information
    = help: add `#![feature(coroutines)]` to the crate attributes to enable
    = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date
-   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`
```
[the remark this answers is not in the document]
Comment (GrigorenkoPV) [reply tests/ui/coroutine/gen_block.none.stderr:76]: Genuinely no idea. Maybe something to do with replacing two equal strings with one Fluent message.
https://github.com/rust-lang/rust/pull/128886/commits/290df4fa5613d3d6cce717509219b18870e3ff86#diff-d93592ecc730e2061ac31cad11e78e3bb7cdc7ca3257a85f04bbd3f48c0c6521L1578-R1589

**Pull request merged**